README.md

```diff
@@ -1,13 +1,51 @@
 # Lab 13
-This branch gives us the major components for creating a manually curated page (albeit a simple one) within PageBuilder. Let's do another exercise to practice what we've learned so far! Switch to `exercise-02` for examples. After completing `exercise-02` return here and we'll start on this branch's user story.
+
+Now that you are a master of content sources, chains, and features, we're going to add a new chain to display features in a more interesting way.
+
+1. Create a new chain called `double-chain`. This chain should split all of its children into two columns. 
+
+2. The user should be able to configure if the column style is `wide-left`, `wide-right`, or `even`. (Use a `PropTypes.oneOf` custom field).
+
+3. The user should also be able to decide the number of features that should exist in the left column. This field should be **required** for display. (use `isRequired`)
+
+4. Now let's fetch content from site service to use in the feature pack. First create a new content source called `site-menu`. 
+
+5. This content source should fetch content from the `/site/v3/navigation/demo/?hierarchy=<hierarchy-name>&_id=$<section-id>` 
+
+6. The content source should take in a `hierarchy` and an `id` parameter. The hierarchy param should use `default` as a default if it is undefined. The id param should use `/` as a default if it is undefined.
+
+7. Check the content PageBuilder debugger. From `http://localhost/pagebuilder/pages` click on `Developer Tools`, `Debugger`, select `Content Debugger`, select `site-menu`, and add test parameters.
+
+## Example Hierarchies from Demo
+- `footer`
+- `main-site-navigation`
+
+## Example Sections from Demo
+- `/food`
+- `/news/arts-and-entertainment`
+- `/news`
+
+Once you've confirmed the source can pull in content, create a schema. (See [Lab 07](https://github.com/wapopartners/Fusion-Training-User-Stories/tree/lab-07) for a refresher on how to create a schema. Also note the script for viewing Fusion data and bookmark it if you haven't yet.)
+
+
+8. Now let's use content from site service to create a navigation bar. Create a new `global` feature called `nav-bar`. 
+
+9. This feature should allow the PageBuilder user to configure the content source so that it takes in the new `site-menu` content source. You'll need a custom field of type `PropTypes.contentConfig('site-menu')`.
+
+10. This featurer should then display the links from site service hierarchy. Use the bookmarked script from Lab 07 to view the data you'll be using.
```

Comment:
what exactly does this script provide that is helpful here?

Reply:
It allows the user to see the data so that they know where to get the links. So they know if it's `content[0].name` or `content.chidren[0].id` or what.

Reply:
It only shows global content though so unless they set it as global content they wouldn't be able to see it. I would have them to use the Content Debugger instead, have they learned about the debugger yet in this lab?